aiida_quantumespresso/workflows/pw/base.py

```diff
@@ -109,6 +115,97 @@ def define(cls, spec):
             message='Then ionic minimization cycle converged but the thresholds are exceeded in the final SCF.')
         # yapf: enable
 
+    @classmethod
+    def get_builder_from_protocol(
+        cls,
+        code,
+        structure,
+        protocol=None,
+        overrides=None,
+        electronic_type=ElectronicType.METAL,
+        spin_type=SpinType.NONE,
+        initial_magnetization=None,
```

Comment:
I named it such in order to match the naming in the common workflows. This would make it easier to use the common workflow generators as well as the built in generators in `aiida-quantumespresso` since all the arguments literally carry over without translation. Do you think that advantage does not outweigh the disadvantage you lined out?

Reply:
Yes, it does! 👍  I was not aware of the naming in the common workflows.

Reply:
Thinking about it some more, I am actually going to rename it to `starting_magnetization` because I just realized that in any case it is not directly equivalent to `initial_magnetization` of the common workflows interface because they have different units. That actually makes it better that they are named different so people won't forget.